src/dune/config.ml

```diff
@@ -206,6 +211,12 @@ let decode =
   and+ cache_check_probability =
     field "cache-check-probablity" Dune_lang.Decoder.float
       ~default:default.cache_check_probability
+  and+ cache_trim_period =
+    field "cache-trim-period" Dune_lang.Decoder.int
```

Comment:
Specifying the period and size with an integer is a bit raw. I propose to accept the following syntaxes:

- for the period: `Ns`, `Nm` and `Nh`
- for the size: `NB`, `NM`and `NG`

Reply:
Bike shed: rather than time, I'd prefer a name like `Duration.t`

Reply:
both suggestions seem good to me!

Reply:
Since they are both `int t`, I went for simple functions, but I can pack them in a module if preferred.